Profile list: clarify which system display profile is preview

flannelhead opened the pull request:
This prevents having a duplicate "system display profile" entry in the histogram profile combobox and clearly indicates which profile is which.

Note that this also renames "system display profile" to "system display profile (preview)" in the "preview display profile" combobox. I think this is beneficial since it may be different from the "system display profile" in the "display profile" combobox (if the preview window is on a different display).

TurboGit reviewed (changes requested):
I don't think "preview" is the right name here as the profile is for the second screen display. I would then use "second screen". We may also want to change the name in dt_colorspaces_get_name() but I'm not sure of all possible effects, so please double check what this will change. TIA.

flannelhead commented:
> I would then use "second screen".

Right, or perhaps "second window" since the tooltip of the button where the second window is launched says "display a second darkroom image window"?

The profile selection popup however calls it the "preview display profile" although it is the very same "second window" that is affected by this combobox.

Just to ensure we're talking about the same thing, it seems to me that `DT_COLORSPACE_DISPLAY2` is used to indicate the profile of the display where the preview window (or second window) is located: https://github.com/darktable-org/darktable/blob/a995e69275da3d85106f10a5814c269d550b8875/src/common/colorspaces.c#L1842-L1853

> We may also want to change the name in dt_colorspaces_get_name() but I'm not sure of all possible effects, so please double check what this will change. TIA.

Alright, going to keep an eye on that. Thanks for the notice.

TurboGit commented:
> Right, or perhaps "second window" since the tooltip of the button where the second window is launched says "display a second darkroom image window"?

Yes, fine by me.

TurboGit commented:
> Just to ensure we're talking about the same thing, it seems to me that `DT_COLORSPACE_DISPLAY2` is used to indicate the profile of the display where the preview window (or second window) is located:

Yes, that's my thinking too even if I've not worked on this part of the code. But I have reviewed it for merging 2 years ago IIRC and the "second window" is really DISPLAY2.

flannelhead commented:
Updated the name to "system display profile (second window)". Also updated `dt_colorspaces_get_name()` - this function seems to be used mainly for debug prints but also for the `colorin` combobox. Anyway both seem supposed to be human-readable (GUI-friendly names) so I don't see a reason not to change both.

TurboGit reviewed (approved):
Looks good, thanks!
